docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -100,8 +167,8 @@ DR paper; IV Iron communications and IDM Symposium)
 
 5.0 References
 ++++++++++++++
-.. [Bauer, 2022]
+.. [Bauer_2022]
```

Comment:
Yep, it was the spaces that was making these not work 👍 And now that you've actually referenced them above, Sphinx won't complain about them being here without being cited.